recipe/deactivate.bat

```diff
@@ -0,0 +1,10 @@
+if defined CONDA_BUILD_STATE (
+    @echo on
+)
+
+if not "%CONDA_CUPY_CUDA_PATH%"=="" (
+    set "%CUDA_PATH%=%CONDA_CUPY_CUDA_PATH%"
+    set "%CONDA_CUPY_CUDA_PATH%="
+) else (
+    set "%CUDA_PATH%="
+)
```

Comment:
This did not restore the environment variable correctly.

```suggestion
if not "%CONDA_CUPY_CUDA_PATH%"=="" (
    set "CUDA_PATH=%CONDA_CUPY_CUDA_PATH%"
    set "CONDA_CUPY_CUDA_PATH="
) else (
    set CUDA_PATH=""
)
```

Reply:
Good catch, thanks!